.github/workflows/email-editor-package.yml

```diff
@@ -106,3 +106,40 @@ jobs:
       - name: Run code style check
         run: ../../../mailpoet/tools/vendor/composer.phar code-style
         working-directory: packages/php/email-editor
+
+  phpstan-static-analysis:
+    runs-on: ubuntu-latest
+
+    strategy:
+      matrix:
+        php-version: ['7.4', '8.2']
+
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v4
+
+      - name: Set up PHP
+        uses: shivammathur/setup-php@v2
+        with:
+          php-version: ${{ matrix.php-version }}
+          extensions: gd
+
+      - name: Install tools
+        run: |
+          COMPOSER_DEV_MODE=1 php tools/install.php
+          touch .env
+        working-directory: mailpoet
+
+      - name: Install composer dependencies
+        run: ../../tools/vendor/composer.phar install
+        working-directory: mailpoet/tasks/phpstan
+
+      - name: Run code phpstan 7.4
+        if: ${{ matrix.php-version == '7.4' }}
+        run: ../../../mailpoet/tools/vendor/composer.phar phpstan -- --php-version=70400
```

Comment:
This would not work. 
I tested this version with another version and got an error.
For example, here is the error for this
<img width="951" alt="Screenshot 2024-11-26 at 8 32 56 AM" src="https://github.com/user-attachments/assets/bc6347a2-ed44-4303-ba66-d64acc6801dd">
<img width="1189" alt="Screenshot 2024-11-26 at 8 36 20 AM" src="https://github.com/user-attachments/assets/72754534-1541-4508-b176-7776062239ae">

Reply:
I asked ChatGPT for this idea, so I apologize for the wrong idea. Because we have an issue with the ignore error related to PHP 8.2 maybe we can create two config files and use `matrix.php-version` as a part of the config name. It is possible to specify the used PHP version in [the configuration](https://phpstan.org/config-reference#phpversion). What do you think about this approach?

Reply:
> Because we have an issue with the ignore error related to PHP 8.2 maybe we can create two config files and use matrix.php-version as a part of the config name. It is possible to specify the used PHP version in [the configuration](https://phpstan.org/config-reference#phpversion). What do you think about this approach?

I don't understand this comment. 

Are you suggesting we create two `email-editor-phpstan.neon` config files?

Reply:
> Are you suggesting we create two email-editor-phpstan.neon config files?

Yes, exactly. I would include the main config, and in the version-specific configs, I would specify the used PHP version.

What do you think about it?

Reply:
Hmmm. That could work. Let me give it a try

Reply:
@lysyjan Instead of creating two `email-editor-phpstan.neon` config files. I'm going to keep the current `email-editor-package.yml` file the same. I believe the way it works is easier to understand and doesn't add an extra overhead.

I will instead update the configuration for `email-editor-phpstan.neon` and allow it to load PHP 7 or PHP 8.2 specific config as it is required.